dev/com.ibm.ws.security.token.ltpa/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -30,3 +30,7 @@ expiration.desc=Amount of time after which a token expires in minutes.
 
 ltpa.monitorInterval=LTPA token keys file polling rate
 ltpa.monitorInterval.desc=Rate at which the server checks for updates to the LTPA token keys file.
+
+authFilterRef=Authentication Filter Reference
+authFilterRef$Ref=Authentication filter reference
+authFilterRef.desc=Specifies the authentication filter reference. 
```

Comment:
@utle In the description, could you be a little more descriptive about what the authentication filter reference is?

Reply:
We have 7 other features mpJwt, saml.sso ...etc. that have the same authFilterRef.desc. Should we keep them the same?

Reply:
Unless the user has to know more this authentication filter reference, then what you have is sufficient.

Reply:
There is a authFilter element that has all the description.

Reply:
I have to close this PR and do a new PR ..